pkg/reconciler.v1/common/job_must_customize.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 The Kubeflow Authors
```

Comment:
Trying to understand name "job_must_customize.go" Is there a better name for some interfaces needed to be implemented?

Reply:
There should be a better name. Maybe we can call it `override.go`. I shall put a readme file right within the `reconciler` package.